### ISSUE TITLE
Frigate addon  will not run

### ISSUE THREAD

**Issue by mwy6283** [blakeblackshear/frigate#972]
**Describe the bug or issue 

 The Frigate addon will not run  please note  in the supervisors  ( highlighted in Yellow log it states contact maintainer of Frigate NVR beta. I noticed this previously i have never used the Beta version so I  delete the addon and download it agains but the same result.

**Version of frigate**
Output from `/api/version i.13`

**Config file**
///mqtt:

  host: 192.168.0.45:1883
  topic_prefix: frigate
  client_id: frigate
  user: mqtt_user
  password: password
  stats-interval: 60

cameras:
  back:
    ffmpeg:
      inputs:
        - path: rtsp://viewer{FRIGATE_RTSP_PASSWORD}@192.168.0.48:554/cam/realmonitor?channel=1&subtype=2
          roles:
            -detect
            -rtmp
    width: 1280
    height: 780
    fps: 5

objects:
  track:
    - person
    - car
    - dog
  filters:
    person:
      min_area: 500
      max_area: 2400000
      min_scaore: 0.5
      threshold: 0.7
      
ffmpeg: 
  hwaccel_args:
    - -hawaccel
    - vappi
    - -hawccel_device
    - /dri/renderD128
    - -hawccel_output_format
    - yuv420p
///

**Frigate container logs**
```
* Starting nginx nginx
   ...done.
Error parsing config: expected a list for dictionary value @ data['cameras']['back']['ffmpeg']['inputs'][0]['roles']```

**Frigate stats**
```json
Output from frigate's /api/stats endpoint
```

**FFprobe from your camera**

Run the following command and paste output below
```
ffprobe <stream_url>
```

**Screenshots**

**Computer Hardware**
 - OS: [e.g. Ubuntu, Windows]
 - Install method: HA-  Frigate Addon & Hacs integration [e.g. Addon, Docker Compose, Docker Command]
 - Virtualization: Virtual Box Linuks on Imac [e.g. Proxmox, Virtualbox]
 - Coral Version: none as yet[e.g. USB, PCIe, None]
 - Network Setup: wired  [e.g. Wired, WiFi]

**Camera Info:**
 - Manufacturer: Reolink[e.g. Dahua]
 - Model:  520[e.g. IPC-HDW5231R-ZE]
 - Resolution: [e.g. 720p]
 - FPS: [e.g. 5]

**Additional context**
 supervisor logs '''21-04-07 13:02:49 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
21-04-07 13:02:49 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
21-04-07 13:02:50 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR
21-04-07 13:02:50 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR Beta (outdated)
21-04-07 13:02:50 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR
21-04-07 13:02:50 WARNING (MainThread) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of Frigate NVR Beta (outdated)
21-04-07 13:02:50 INFO (MainThread) [supervisor.store] Loading add-ons from store: 67 all - 0 new - 0 remove
21-04-07 13:02:50 INFO (MainThread) [supervisor.store] Loading add-ons from store: 67 all - 0 new - 0 remove
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.check] System checks complete
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
21-04-07 14:02:49 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
21-04-07 14:02:49 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
21-04-07 14:16:10 INFO (MainThread) [supervisor.host.info] Updating local host information
21-04-07 14:16:10 INFO (MainThread) [supervisor.host.services] Updating service information
21-04-07 14:16:10 INFO (MainThread) [supervisor.host.network] Updating local network information
21-04-07 14:16:11 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
21-04-07 14:16:11 INFO (MainThread) [supervisor.host] Host information reload completed
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.check] System checks complete
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
21-04-07 15:02:49 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete'''

Screenshot integration  error 
![Screenshot 2021-04-04 at 08 28 08](https://user-images.githubusercontent.com/57802561/113895689-1f078180-97c1-11eb-8ece-b88f088e7b35.png)

**Comment by kbmetz** [comment]
Frigate isn't running. See the container log

Error parsing config: expected a list for dictionary value @ data['cameras']['back']['ffmpeg']['inputs'][0]['roles']```

Fix your config.yaml and then run the container again. Ensure there are no errors

**Comment by heikkis** [comment]
Role sector is missing space between "-" and the value.

```
roles:
-detect
-rtmp
```

=>

```
roles:
- detect
- rtmp
```

**Comment by mrmarria** [comment]
Just how and where does one do this:
FFprobe from your camera

Run the following command and paste output below

ffprobe <stream_url>

**Comment by NickM-27** [comment]
> Just how and where does one do this:
> 
> FFprobe from your camera
> 
> 
> 
> Run the following command and paste output below
> 
> 
> 
> ffprobe <stream_url>
> 
> 

You'll have to run it manually from the CLI, in the next version frigate will have a button in the UI to do it for each camera.

**Comment by mrmarria** [comment]
Quick response appreciated, but seems to be an unknown command in the 
pi4 HA core...

**Comment by NickM-27** [comment]
> Quick response appreciated, but seems to be an unknown command in the pi4 HA core...

yes, you have to run it inside frigate `docker exec -it addon_ccab4aaf_frigate-fa ffprobe ...` (need to change the name based on which addon you have installed, and add the camera stream in place of ...)
